repository/gorm/channel.go

```diff
@@ -240,6 +240,47 @@ func (repo *Repository) GetDirectMessageChannelMapping(userID uuid.UUID) (mappin
 		Error
 }
 
+// GetDirectMessageChannelList implements ChannelRepository interface.
+func (repo *Repository) GetDirectMessageChannelList(userID uuid.UUID) (dmChannels []model.DMChannel, err error) {
+	dmChannelMapping := make([]*model.DMChannelMapping, 0)
+	if userID == uuid.Nil {
+		return dmChannels, nil
+	}
```

Comment:
ここは適切にエラーを返したほうがいいと思います
ちょうどエラーの値が package 内部で用意されているので、これを使うといいと思います！
```suggestion
	if userID == uuid.Nil {
		return dmChannels, repository.ErrNilID
	}
```